helm/charts/hydra/templates/job-migrations-custom.yaml

```diff
@@ -0,0 +1,132 @@
+{{- range $jobName, $job := $.Values.hydra.customMigrations.jobs }}
+{{- if $job.enabled -}}
+{{- $nodeSelector := ternary $job.nodeSelector $.Values.job.nodeSelector (not (empty $job.nodeSelector)) -}}
+{{- $migrationExtraEnv := ternary $job.extraEnv $.Values.job.extraEnv (not (empty $job.extraEnv)) -}}
+{{- $resources := ternary $job.resources $.Values.job.resources (not (empty $job.resources)) -}}
+{{- $annotations := merge $.Values.job.annotations (default dict $job.annotations) -}}
+{{- $labels := merge $.Values.job.labels (default dict $job.labels) -}}
+
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "hydra.fullname" $ }}-{{ $jobName }}
+  {{- if $.Release.Namespace }}
+  namespace: {{ $.Release.Namespace }}
+  {{- end }}
+  labels:
+    {{- include "hydra.labels" $ | nindent 4 }}
+    {{- with $labels }}
+      {{- toYaml . | nindent 4 }}
+    {{- end }}
+  annotations:
+    {{- with $annotations }}
+      {{- toYaml . | nindent 4 }}
+    {{- end }}
+spec:
+  template:
+    metadata:
+      annotations:
+        {{- with $annotations }}
+          {{- toYaml . | nindent 8 }}
+        {{- end }}
+        {{- with $.Values.job.podMetadata.annotations }}
+          {{- toYaml . | nindent 8 }}
+        {{- end }}
+      labels:
+        app.kubernetes.io/name: {{ include "hydra.fullname" $ }}-{{ $jobName }}-automigrate
+        app.kubernetes.io/instance: {{ $.Release.Name }}
+        {{- with $labels }}
+          {{- toYaml . | nindent 8 }}
+        {{- end }}
+        {{- with $.Values.job.podMetadata.labels }}
+          {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      {{- with $.Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "hydra.job.serviceAccountName" $ }}
+      automountServiceAccountToken: {{ $.Values.job.automountServiceAccountToken }}
+      containers:
+        - name: {{ $.Chart.Name }}-{{ $jobName }}
+          image: "{{ $.Values.image.repository }}:{{ $.Values.image.tag }}"
+          imagePullPolicy: {{ $.Values.image.pullPolicy }}
+        {{- if $job.customCommand }}
+          command: {{- toYaml $job.customCommand | nindent 10 }}
+        {{- else }}
+          command: ["hydra"]
+        {{- end }}
+        {{- if $job.customArgs }}
+          args: {{- toYaml $job.customArgs | nindent 10 }}
+        {{- else }}
+          args: ["migrate", "sql", "-e", "--yes", "--config", "/etc/config/hydra.yaml"]
+        {{- end }}
```

Comment:
for custom job the command and args should be implicit without fallback to default migration - no need to run twice same ops